### PR TITLE
Fixing elastic curator, elasticsearch, fluentd

### PR DESCRIPTION
**Pull request by lieberlois**

### CODE REVIEW

File: stack/kubernetes/tools/logging/fluentd.yaml [hunk 1]
```diff
@@ -1,7 +1,11 @@
+podSecurityPolicy:
```
Comment (BlacCello) [inline stack/kubernetes/tools/logging/fluentd.yaml:1]: Please also adjust `local/kubernetes/tools/logging/fluentd.yaml` 
Side note: I don't want to have all those files replicated, but currently we have this setup...

File: stack/kubernetes/24_deploy_logging.sh [hunk 2]
```diff
@@ -3,6 +3,7 @@ set -euo pipefail
 
 NAMESPACE=logging
 ELASTIC_VERSION="7.17.3"
+FLUENTD_VERSION="0.4.3"
```
Comment (BlacCello) [inline stack/kubernetes/24_deploy_logging.sh:6]: Please adjust `local/kubernetes/22b_deploy_logging_elastic.sh` as well.

File: stack/kubernetes/tools/logging/curator.yaml [hunk 3]
```diff
@@ -1,4 +1,7 @@
 ---
+image:
```
Comment (BlacCello) [inline stack/kubernetes/tools/logging/curator.yaml:2]: local file has to be adjusted as well

File: stack/kubernetes/tools/logging/elasticsearch.yaml [hunk 4]
```diff
@@ -1,21 +1,27 @@
 ---
```
Comment (BlacCello) [inline stack/kubernetes/tools/logging/elasticsearch.yaml:1]: local file has to be adjusted as well

**Review by BlacCello — CHANGES_REQUESTED** [review]

File: stack/kubernetes/tools/logging/fluentd.yaml (hunk 1 above)
Comment (lieberlois) [reply stack/kubernetes/tools/logging/fluentd.yaml:1]: I added the new prometheus source, the PSP was already set to disabled in the local file.

File: stack/kubernetes/tools/logging/elasticsearch.yaml (hunk 4 above)
Comment (lieberlois) [reply stack/kubernetes/tools/logging/elasticsearch.yaml:1]: Done :) i set the RAM requests to 1 Gi (JVM has 512 MB, which is recommended for Elastic / OpenSearch - 50 % of available RAM for the JVM memory)

File: stack/kubernetes/tools/logging/curator.yaml (hunk 3 above)
Comment (lieberlois) [reply stack/kubernetes/tools/logging/curator.yaml:2]: Done.

File: stack/kubernetes/24_deploy_logging.sh (hunk 2 above)
Comment (lieberlois) [reply stack/kubernetes/24_deploy_logging.sh:6]: Done

**Review by BlacCello — APPROVED** [review]

**Pull request merged**